### PR TITLE
[stable/prometheus-operator] Bump operator version. Closes #20503

### LINKED ISSUE #20503: [stable/prometheus-operator] podMetadata annotations not being propagated to Prometheus instance pod
**Describe the bug**
I need to add an annotation to the running prometheus instance. I used the helm chart as follows (unnessecary information ommited):
```
prometheus:
  enabled: true
  prometheusSpec:
  ...
    podMetadata:
      annotations:
        backup.velero.io/backup-volumes: prometheus-prometheus-federated-tst-p-prometheus-db
  ...
```

Unfortunately this annotation is not set after creating the prometheus pod:
```
kubectl -n prometheus-federated-tst describe pod prometheus-prometheus-federated-tst-p-prometheus-0
Name:           prometheus-prometheus-federated-tst-p-prometheus-0
Namespace:      prometheus-federated-tst
Priority:       0
Node:           *********/**.**.**.**
Start Time:     Mon, 03 Feb 2020 13:36:23 +0100
Labels:         app=prometheus
                controller-revision-hash=prometheus-prometheus-federated-tst-p-prometheus-bdb9bd8f5
                prometheus=prometheus-federated-tst-p-prometheus
                statefulset.kubernetes.io/pod-name=prometheus-prometheus-federated-tst-p-prometheus-0
Annotations:    kubernetes.io/limit-ranger:
                  LimitRanger plugin set: cpu, memory request for container prometheus-config-reloader; cpu, memory limit for container prometheus-config-re...
                kubernetes.io/psp: prometheus-federated-tst-p-prometheus
Status:         Running
...
```

**Version of Helm and Kubernetes**:
Kubernetes 1.15
Helm  v2.12.0

**Which chart**:
Chart: stable/prometheus-operator 
Version:  8.6.0

**What happened**:
Annotation missing.

**What you expected to happen**:
Existing annotation on pod

**How to reproduce it** (as minimally and precisely as possible):

**Anything else we need to know**:
Regarding to issue #16272 it should be working.

### PR DESCRIPTION
**Pull request by jbfavre**
Signed-off-by: Jean Baptiste Favre <github@jbfavre.org>

#### What this PR does / why we need it:
This PR bumps operator version to use latest release.

#### Which issue this PR fixes
  - fixes #20503 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Review by Vad1mo — APPROVED** [review]

File: stable/prometheus-operator/values.yaml [hunk 1]
```diff
@@ -1184,7 +1184,7 @@ prometheusOperator:
   ##
   image:
     repository: quay.io/coreos/prometheus-operator
-    tag: v0.36.0
+    tag: v0.37.0
```
Comment (davidkarlsen) [inline stable/prometheus-operator/values.yaml:1187]: please make readme match too

Comment (jbfavre) [reply stable/prometheus-operator/values.yaml:1187]: Done, thanks for the catch

**Pull request merged**